docs/data/migration/migration-pickers-v6/migration-pickers-v6.md

```diff
@@ -125,7 +125,24 @@ To keep the same behavior, you can replace it by `hasLeadingZerosInFormat`
  );
 ```
 
-## Removed formats
+## Date management
+
+### Use localized week with luxon
+
+The Date and Time Pickers now use the localized week when using `AdapterLuxon` and Luxon v3.4.4 or higher is installed.
```

Comment:
can we add this to the `peerDependencies` maybe?

Reply:
We are still supporting Luxon < 3.4.4, if we change the peer deps we would force a more recent version than needed.